test/updateDB.test.js

```diff
@@ -0,0 +1,197 @@
+const path = require('path')
```

Comment:
We use dashed file names like [`update-db.test.js`](https://github.com/browserslist/browserslist/blob/master/test/shareable-stats.test.js)